x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/dashboards/entity_analytics.cy.ts

```diff
@@ -356,18 +349,16 @@ describe('Entity Analytics Dashboard', { tags: ['@ess', '@serverless'] }, () =>
     describe('With host risk data', () => {
       before(() => {
         cy.task('esArchiverLoad', { archiveName: 'risk_scores_new' });
-        login();
-        enableRiskEngine();
       });
 
       beforeEach(() => {
         login();
+        mockRiskEngineEnabled();
```

Comment:
seems like we do this for each `describe` fixture. I think there's a `beforeEach` that applies to to all fixtures. Is it possible to move  `login` and `mockRiskEngine` there?